packages/webapp/package.json

```diff
@@ -35,6 +35,7 @@
     "websocket": "1.0.34"
   },
   "devDependencies": {
+    "@svgr/webpack": "^6.2.1",
```

Comment:
see my comment in the webpack config

Reply:
Thanks for your suggestion @martindevtum 👍

I did get rid of this package, but with this package being gone, [we will not be able to style the SVG dynamically based on the external CSS file](https://stackoverflow.com/a/20720935) (which is needed for changing the icon of theme switcher). 

So as of now, I think our possible approaches would be:
-  to either use this package (if we want to define svg in a file), or
-  use inline svg instead 

Please feel free to suggest otherwise if I am missing something. (: